src/actor/Verifier.ts

```diff
@@ -215,26 +239,24 @@ export async function verifyPresentation(
   // if we got a privacy enhanced attestation, check that this was allowed by the verifier and
   // verify the attestation
   if (message.body.type === MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES_PE) {
-    if (
-      typeof latestAccumulators === 'undefined' ||
-      typeof attesterPubKeys === 'undefined'
-    ) {
-      throw new Error(`Received privacy enhanced presentation. Require:
-      Session: ${session}
-      accumulators: ${latestAccumulators}
-      public keys: ${attesterPubKeys}`)
+    const accFailure = !verificationCheck(_latestAccumulators, true)
```

Comment:
Note: This function fixes the case in which any of `latestAccumulators` or `attesterPubKeys` are anything but `undefined`